code/datums/components/nanites.dm

```diff
@@ -0,0 +1,163 @@
+/datum/component/nanites
+	dupe_mode = COMPONENT_DUPE_UNIQUE_PASSARGS
+
+	var/mob/living/host_mob
```

Comment:
don't be saving pre type pathed things like this. Check to make sure you arent an incompatible type in init then you can assume you aren't somewhere weird elsewhere and simply typepath `var/mob/living/master = parent` in the proc

Reply:
A potential future feature would involve nanites stored in clothing or other forms of storage, whose effects apply to the mob that wears them; using a separate host_mob i don't hard set it to the parent